### PR TITLE
chore(repo): use issue forms, feat(website): link to subreddit

### PR DESCRIPTION
**Pull request by l-zeuch**
**Please describe the changes this PR makes and why it should be merged:**

This pull request reworks the issue templates to use the beta functionality of forms instead.
Seeing as forms have the ability of required fields plus are much easier to use, this might only be beneficial.

Additionally, this will also add the subreddit to the website's footer, as I wanted to add that in my last pull request but kind of forgot about it.

**Status**

- [x] Code changes have been tested on an instance of YAGPDB, or there are no code changes
- [x] I have read and followed the [contribution guide](../CONTRIBUTING.md)
- [x] I have [written documentation](../WRITING-DOCUMENTATION.md) for my changes, or there is no need to

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    attributes:
+      label: Solution
+      description: Explain the solution you would like to see. Please also provide alternatives to your solution.
+    validations:
+      required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduc(../CODE_OF_CONDUCT.md)
```
Comment (jo3-l) [inline .github/ISSUE_TEMPLATE/feature_request.yml:27]:
```suggestion
      description: By submitting this issue, you agree to follow our [Code of Conduct](../CODE_OF_CONDUCT.md)
```
:O

**Review by jo3-l — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (l-zeuch) [reply .github/ISSUE_TEMPLATE/feature_request.yml:27]: oh noes

**Comment by jo3-l** [comment]
Are you certain the issue forms are valid? Asking 'cuz I see this when I try to preview:
image
Could just be GitHub acting up but better to double check :P

**Comment by l-zeuch** [comment]
Checking with [this extension](https://github.com/redhat-developer/vscode-yaml) and the correct scheme I get no errors...
Also comparing it to working forms shows no mistakes.

**Comment by jo3-l** [comment]
Alright, in that case, let's :shipit:

**Review by jo3-l — APPROVED** [review]

**Comment by l-zeuch** [comment]
:shipit:

**Pull request merged**